streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -1798,8 +1746,6 @@ public void shouldCloseStateManagerEvenDuringFailureOnUncleanTaskClose() {
 
         assertThrows(RuntimeException.class, () -> task.suspend());
         task.closeDirty();
-
-        EasyMock.verify(stateManager);
```

Comment:
Verifications are missing in the new code. For example, `stateManager.close()`.

Reply:
I have moved
```
doNothing().when(stateManager).close();
```
to
```
verify(stateManager).close();
```
in subsequent commits.
Everything else is related to empty maps or sets, so I have removed the stubbings